### PR TITLE
Add moinmoin

### PR DESCRIPTION
**Pull request by jhermann**
Similarweb rank: open source logo, about 615.000 Google results for "MoinMoin wiki"

Preview: https://github.com/jhermann/simple-icons/blob/add-moinmoin/icons/moinmoin.svg

Checklist

 * [x] updated the JSON data in _data/simple-icons.json
 * [x] optimized the icon with SVGO or SVGOMG
 * [x] The SVG viewbox is 0 0 24 24

Description

 *  Why did you pick the hex value? Sky blue of original logo
 *  Did you manually vectorize the logo? Original SVG
 *  Have you used multiple sources? Original logo

### CODE REVIEW

**Comment by jhermann** [comment]
I have no idea what to do abou those failed checks.

**Comment by LitoMore** [comment]
@jhermann Please kindly read through our [contributing documentation](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md). Especially the [Annotate the icon](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#4-annotate-the-icon) and [Check the icon](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#5-check-the-icon) section.

**Comment by service-paradis** [comment]
@jhermann Can you do a quick Google Trends comparison with similar tools we already added in the repo? Right now, it is out of scope.

File: icons/moinmoin.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...045V10.95a2.67 2.67 0 0 0 .964-2.043 2.703 2.703 0 0 0-2.707-2.69A2.7 2.7 0 0 0 14.7 8.428l-1.71 1.066a2.693 2.693 0 0 0-.933-.175c-.325 0-.632.066-.92.17L9.412 8.417a2.7 2.7 0 0 0-2.656-2.2Zm0 .414c1.157 0 2.11.847 2.27 1.948.008.06.043.113.094.145l1.875 1.165a.207.207 0 0 0 .188.017 2.302 2.302 0 0 1 1.753.004.207.207 0 0 0 .19-.017l1.864-1.159a.207.207 0 0 0 .095-.146 2.286 2.286 0 0 1 2.272-1.957 2.282 2.282 0 0 1 2.293 2.276c0 .73-.347 1.375-.885 1.793-.05.039-.08.1-.08.164v2.325c0 .064.03.125.08.164.538.417.885 1.065.885 1.795a2.282 2.282 0 0 1-2.293 2.276 2.284 2.284 0 0 1-2.296-2.276c0-.787.4-1.48 1.012-1.89a.207.207 0 0 0 .093-.172v-1.741a.207.207 0 0 0-.317-.175l-1.411.876a.207.207 0 0 0-.097.157 2.284 2.284 0 0 1-2.285 2.082 2.286 2.286 0 0 1-2.286-2.09.207.207 0 0 0-.097-.158L8.23 11.14a.207.207 0 0 0-.317.174v1.752c0 .07.036.135.095.173a2.273 2.273 0 0 1 1.044 1.909 2.284 2.284 0 0 1-2.295 2.276 2.282 2.282 0 0 1-2.293-2.276c0-.715.33-1.35.85-1.767a.207.207 0 0 0 .078-.162v-2.383a.207.207 0 0 0-.078-.162 2.26 2.26 0 0 1-.85-1.767 2.282 2.282 0 0 1 2.293-2.276z" style="opacity:0.99;fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:0.0690307"/></svg>
```
Comment (PratyushJayachandran) [inline icons/moinmoin.svg:1]: You need to merge and join all paths before optimizing so that the resulting SVG matches the guidelines.

**Review by PratyushJayachandran — COMMENTED** [review]
Thank you for staring the PR.
Since, it is not within our threshold of Similarweb ranking, please provide us with some reasons as to why this brand is popular enough to be added.

**Comment by PeterShaggyNoble** [comment]
Closing this as it is now ranked at 1.2m, putting it _well_ outside our scope. @jhermann, if you _can_ provide an alternative metric of popularity such as those suggested above then we can revisit this.

**Pull request closed without merging**